### DISCUSSION TITLE
Portable units

### DISCUSSION THREAD

**Discussion by dc1rdb** [OpenGammaProject/Open-Gamma-Detector#39] [Show and tell]
Here are two portable units of the Open Gamma Detector I built recently. Of course, they also can be used stationary connected to the Gamma MCA app via USB.
The main difference is the size of the NaI(Tl) scintillator. The bigger unit utilizes a 1" dia x 2" long and on the smaller one I've used a 1" dia x 1" long.
Both scintillators are coupled to a 2x2 SiPM matrix, which makes quite a difference to a single SiPM.

![IMG_20230607_193508](https://github.com/OpenGammaProject/Open-Gamma-Detector/assets/9314230/4e98c398-7da1-4892-9dbd-3e2f7c03a598)
![IMG_20230607_180517](https://github.com/OpenGammaProject/Open-Gamma-Detector/assets/9314230/f614a85d-7548-498f-884f-0061c9d1af4e)
![IMG_20230611_191933](https://github.com/OpenGammaProject/Open-Gamma-Detector/assets/9314230/da55d4ca-04eb-43d0-a09f-ed44ea50da95)
![IMG_20230517_193242](https://github.com/OpenGammaProject/Open-Gamma-Detector/assets/9314230/403ee1d8-a2e0-4f69-88e0-7583f4c6b979)

**Comment by NuclearPhoenixx** [comment]
Awesome cases together with the 3D printed parts! I hope you will enjoy it for a long time 🙂
